marketplace-and-integrations/integrations/algolia.md

```diff
@@ -91,8 +89,15 @@ The list of default properties consists of:
 * `ObjectID` - GUID`from the content item's`Key\` property
 * `Name` - with culture variants if any
 * `CreateDate`
+* `CreatorName`
 * `UpdateDate`
+* `WriterName`
+* `TemplateId`
+* `Level`
+* `Path`
+* `ContentTypeAlias`
 * `Url` - with culture variants if any
+* registered properties of the content type
```

Comment:
Small suggestions to this bullet.
Also, shouldn't it be Document Type instead of Content Type? 🤔 

```suggestion
* Any registered properties on the Content Type
```

Reply:
Made the update, thank you @sofietoft